podman/domain/containers.py

```diff
@@ -472,6 +472,10 @@ def stop(self, **kwargs) -> None:
         if response.status_code == requests.codes.not_modified:
             if kwargs.get("ignore", False):
                 return
+            else:
+                raise APIError(
+                    response.text, response=response, explanation="Container already stopped."
```

Comment:
would you consider populating the response with `response.json()["message"]` rather than a hard coded explanation?

Reply:
IIRC The problem this PR attempted to solve was that podman compose tried to stop stopped container. There was no json-parseable body in the response and therefore the explanation couldn't be populated from the json in that case.

I found a place in the integration tests where I can stop the stopped container and expect the APIError. I've added the test in preceeding commit so we can see how it fails. Perhaps the fix should in the API -- to always produce json-parseable body.

Thank you for suggestion to write the tests. Is there some manual to help me running the tests? @inknos